### PR TITLE
Stage2 collections

### PR DESCRIPTION
**Pull request by KristinaPotapovich**

### CODE REVIEW

File: stage #2/module #3 Collections/lesson #4. Set/set.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+```
+
+
+* Contains unique elements only like HashSet. It extends the HashSet class and implements the Set interface.
+* Maintains insertion order.
+
+Let's see a simple example of Java LinkedHashSet class.
+```
+import java.util.*;
```
Comment (BogdanovaDiana) [inline stage #2/module #3 Collections/lesson #4. Set/set.md:90]: Please avoid using wildcard imports

File: stage #2/module #3 Collections/lesson #4. Set/set.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+* Overview
+* HashSet
+* LinkedHashSet
+* TreeSet
+* Materials
+
+## Overview
+The Set interface is present in java.util package and extends the Collection interface is an unordered collection of objects in which duplicate values cannot be stored. It is an interface that implements the mathematical set. This interface contains the methods inherited from the Collection interface and adds a feature that restricts the insertion of the duplicate elements. There are two interfaces that extend the set implementation namely SortedSet and NavigableSet.
+
+![](./media/unnamed.png)
```
[the remark this answers is not in the document]
Comment (KristinaPotapovich) [reply stage #2/module #3 Collections/lesson #4. Set/set.md]: fixed

File: stage #2/module #3 Collections/lesson #4. Set/set.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+import java.util.*;  
+class HashSet1{  
+ public static void main(String args[]){  
+  //Creating HashSet and adding elements  
+    HashSet<String> set=new HashSet();  
+           set.add("One");    
+           set.add("Two");    
+           set.add("Three");   
+           set.add("Four");  
+           set.add("Five");
```
[the remark this answers is not in the document]
Comment (KristinaPotapovich) [reply stage #2/module #3 Collections/lesson #4. Set/set.md]: fixed

File: stage #2/module #3 Collections/lesson #4. Set/set.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+               while(i.hasNext())  
+               {  
+               System.out.println(i.next());  
+               }  
+ }  
+}  
+```
+Following is the difference between LinkedHashMap and LinkedHashSet:
+
+![](./media/Capture-70.jpg)
```
[the remark this answers is not in the document]
Comment (KristinaPotapovich) [reply stage #2/module #3 Collections/lesson #4. Set/set.md]: fixed

File: stage #2/module #3 Collections/lesson #4. Set/set.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+
+Declaration of HashSet:
+```
+public class HashSet<E> extends AbstractSet<E> implements Set<E>, Cloneable, Serializable
+```
+where E is the type of elements stored in a HashSet.
+
+Let's see a simple example of HashSet.
+```
+import java.util.*;
```
[the remark this answers is not in the document]
Comment (KristinaPotapovich) [reply stage #2/module #3 Collections/lesson #4. Set/set.md]: fixed

File: stage #2/module #3 Collections/lesson #4. Set/set.md (hunk 1 above)
Comment (KristinaPotapovich) [reply stage #2/module #3 Collections/lesson #4. Set/set.md]: fixed

File: stage #2/module #3 Collections/lesson #4. Set/set.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+=================================================
+## Materials
+* Overview
+* HashSet
+* LinkedHashSet
+* TreeSet
+* Materials
+
+## Overview
+The Set interface is present in java.util package and extends the Collection interface is an unordered collection of objects in which duplicate values cannot be stored. It is an interface that implements the mathematical set. This interface contains the methods inherited from the Collection interface and adds a feature that restricts the insertion of the duplicate elements. There are two interfaces that extend the set implementation namely SortedSet and NavigableSet.
```
[the remark this answers is not in the document]
Comment (KristinaPotapovich) [reply stage #2/module #3 Collections/lesson #4. Set/set.md:11]: fixed

**Pull request merged**